### PR TITLE
Add tracing-tracy to related crates

### PR DESCRIPTION
**Pull request by nagisa**
## Motivation

tracing-tracy has become tested enough that I’m comfortable with suggesting it to
people at least to try out.

## Solution

List the crate among other related crates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -397,6 +397,7 @@ are not maintained by the `tokio` project. These include:
   environment variables with an `env_logger` compatible syntax.
 - [`tracing-unwrap`] provides convenience methods to report failed unwraps on `Result` or `Option` types to a `Subscriber`.
 - [`diesel-tracing`] provides integration with [`diesel`] database connections.
+- [`tracing-tracy`] provides a quick way to collect Tracy profiles in instrumented applications.
```
Comment (davidbarsky) [inline README.md:400]:
```suggestion
- [`tracing-tracy`] provides a way to collect [Tracy] profiles in instrumented applications.
```

File: README.md [hunk 2]
```diff
@@ -422,6 +423,7 @@ please let us know!)
 [`tracing-unwrap`]: https://docs.rs/tracing-unwrap
 [`diesel`]: https://crates.io/crates/diesel
 [`diesel-tracing`]: https://crates.io/crates/diesel-tracing
+[`tracing-tracy`]: https://crates.io/crates/tracing-tracy
```
Comment (davidbarsky) [inline README.md:427]:
```suggestion
[`diesel`]: https://crates.io/crates/diesel
[`diesel-tracing`]: https://crates.io/crates/diesel-tracing
[`tracing-tracy`]: https://crates.io/crates/tracing-tracy
[Tracy]: https://github.com/wolfpld/tracy
```

File: tracing/src/lib.rs [hunk 3]
```diff
@@ -729,6 +729,8 @@
 //!  - [`tracing-unwrap`] provides convenience methods to report failed unwraps
 //!    on `Result` or `Option` types to a `Subscriber`.
 //!  - [`diesel-tracing`] provides integration with [`diesel`] database connections.
+//!  - [`tracing-tracy`] provides a quick way to collect Tracy profiles in instrumented
+//!    applications.
```
Comment (davidbarsky) [inline tracing/src/lib.rs:733]:
```suggestion
//! - [`tracing-tracy`] provides a way to collect [Tracy] profiles in 
//!    instrumented applications.
```

File: tracing/src/lib.rs [hunk 4]
```diff
@@ -752,6 +754,7 @@
 //! [`tracing-unwrap`]: https://docs.rs/tracing-unwrap
 //! [`diesel`]: https://crates.io/crates/diesel
 //! [`diesel-tracing`]: https://crates.io/crates/diesel-tracing
+//! [`tracing-tracy`]: https://crates.io/crates/tracing-tracy
 //!
```
Comment (davidbarsky) [inline tracing/src/lib.rs:758]:
```suggestion
//! [`tracing-tracy`]: https://crates.io/crates/tracing-tracy
//! [Tracy]: https://github.com/wolfpld/tracy
//! 
```

**Review by davidbarsky — CHANGES_REQUESTED** [review]

**Comment by davidbarsky** [comment]
Thanks for opening this, btw!

**Comment by hawkw** [comment]
Can we also add a link to the list of related crates [here](https://github.com/tokio-rs/tracing/blob/master/tracing/README.md#related-crates)?

**Comment by nagisa** [comment]
Comments addressed.

**Review by hawkw — APPROVED** [review]
looks good to me! thanks!

**Pull request merged**